Codeinwp/neve
# Pro/feat/otter install module dashboard

### Summary

This PR is a helper PR of https://github.com/Codeinwp/neve-pro-addon/pull/2282

*  Install/Activate plugins mechanism of the StarterSitesUnavailable.js component has been transformed into a re-usable component. 
* Activate/Install dependent plugins on Dashboard Module Cards capability added. For now; only Otter/Otter Pro in Block Editor is supported.

### Will affect visual aspect of the product

YES

### Screenshots 

### Test instructions

Check instructions at https://github.com/Codeinwp/neve-pro-addon/pull/2282


Closes https://github.com/Codeinwp/neve-pro-addon/issues/2222.
